azure-pipelines.yml

```diff
@@ -218,18 +220,12 @@ stages:
           configuration: '$(buildConfiguration)'
           msbuildArgs: '/p:DeployExtension=false /p:Sha1=$(Build.SourceVersion) /p:BuildNumber=$(Build.BuildId)'
 
-      - task: VSTest@2
-        displayName: "Run SonarAnalyzer unit tests"
-        inputs:
-          testAssemblyVer2: |
-            **\$(BuildConfiguration)\**\*.UnitTest.dll
-            !**\obj\**
-          runInParallel: true
-          runTestsInIsolation: true
-          codeCoverageEnabled: true
-          platform: '$(buildPlatform)'
-          configuration: '$(buildConfiguration)'
-          rerunFailedTests: false
+      - powershell: |
+          cd sonaranalyzer-dotnet\tests
+          $(NUGET_PACKAGES)\opencover\4.7.922\tools\OpenCover.Console.exe -target:"C:/Program Files/dotnet/dotnet.exe" -targetargs:"test SonarAnalyzer.UnitTest\bin\$(BuildConfiguration)\net48\SonarAnalyzer.UnitTest.dll" -filter:"+[SonarAnalyzer.CFG]* +[SonarAnalyzer]* +[SonarAnalyzer.CSharp]* +[SonarAnalyzer.Utilities]* +[SonarAnalyzer.VisualBasic]*" -output:coverage.xml -oldStyle -register:Path32
```

Comment:
you may want to use `NugetPackageRoot` 

see https://github.com/SonarSource/sonar-dotnet/pull/3226 and https://github.com/NuGet/Home/issues/6301#issuecomment-350939781

Reply:
Isn't this strictly related to MsBuild?

Reply:
Hmm I believe you are correct. That would mean we need to put it back in our Readme?

Reply:
I don't think we need to add it in the Readme. This is used only by the build script.